setup.py

```diff
@@ -0,0 +1,31 @@
+import os
+import setuptools
+
+with open("README.md", "r") as readme:
+    long_description = readme.read()
+
+deps = os.popen("pip freeze").readlines()
+# readlines() doesn't strip trailing newlines
+deps = [d.strip() for d in deps]
+
+setuptools.setup(
+    name="jigsaw-net-analysis",
+    version="0.0.1",
+    author="Jigsaw Operations, LLC",
+    author_email="fortuna@google.com",
+    description="Network analysis tools",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    url="https://github.com/Jigsaw-Code/net-analysis",
+    packages=setuptools.find_packages(),
```

Comment:
I don't think this is working. Maybe because there are not `__init__.py` files. Though, AFAIK, Python 3 doesn't need them.

You can also explicitly list them: `packages=["netanalysis", "third_party"]`

Reply:
Oh, you should use `setuptools.find_namespace_packages()` instead.

Reply:
I see you are adding `__init__.py` files. Python 3 doesn't need them, so I'd rather not add them since they are overhead. Did `setuptools.find_namespace_packages()` not work?

Reply:
I take that back. I think it will be useful to have the `__init__.py` files, since I want to mix different languages in this repo (already have shell, for example). Also, that still seems to be recommended if it's not a namespace package. You can go ahead with what you're doing.